xbmc/pvr/PVRManager.cpp

```diff
@@ -800,9 +800,6 @@ void CPVRManager::OnPlaybackStopped(const CFileItemPtr item)
   {
     UpdateLastWatched(item->GetPVRChannelInfoTag());
 
-    // store channel settings
-    g_application.SaveFileState();
```

Comment:
@FernetMenta Yes, finally this can go. Unfortunately, I moved this with my latest PR to `CPVRGUIActions::OnPlaybackStopped` to get rid of PVR Manager's dependency to `g_application`. So, could you please remove the respective call from  `CPVRGUIActions::OnPlaybackStopped` to make Kodi compile again?

Reply:
I did while you were writing this :)